code/game/atoms.dm

```diff
@@ -513,6 +522,8 @@
 		var/mob/M = mob
 		if(M.see_invisible >= invisibility && MOB_CAN_SEE_PLANE(M, plane))
 			M.show_message(message, VISIBLE_MESSAGE, blind_message, AUDIBLE_MESSAGE)
+			if(runemessage != -1)
+				M.create_chat_message(src, "* [runemessage || message] *", FALSE, list("emote"), audible = FALSE)

```

Comment:
I don't think this works correctly, there doesn't appear to be any condition where `message` is possible to print. I assume this should just work if you don't specify runemessage and instead right now it'll print just an eye.

Reply:
Legacy, I'll remove.